src/com/haskforce/codeInsight/HaskellCompletionCacheLoader.scala

```diff
@@ -7,38 +7,26 @@ import com.haskforce.psi.{HaskellFile, HaskellPsiUtil}
 import com.intellij.AppTopics
 import com.intellij.codeInsight.lookup.LookupElement
 import com.intellij.openapi.application.ApplicationManager
-import com.intellij.openapi.components.ProjectComponent
 import com.intellij.openapi.editor.Document
 import com.intellij.openapi.fileEditor._
-import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.{Project, ProjectManagerListener}
 import com.intellij.openapi.util.Computable
 import com.intellij.openapi.vfs._
 import com.intellij.psi.{PsiFile, PsiManager}
 
 import scala.annotation.tailrec
 
 /** Loads the completion cache for any Haskell files we have open. */
-class HaskellCompletionCacheLoader(project: Project) extends ProjectComponent {
+class HaskellCompletionCacheLoader extends ProjectManagerListener {
```

Comment:
I think this might require changes to the plugin.xml?

Reply:
Yes, I forgot to register the ApplicationInitialized instance. I'll add that